config/sidekiq.yml

```diff
@@ -1,4 +1,7 @@
 :schedule:
   daily_digest:
-    cron: '0 */15 * * *'
+    cron: '0 15/24 * * *'
     class: DailyDigestWorker
+  sphinx_indexer:
```

Comment:
Do we need reindexing via Cron?
Can we do it automatically after updating the record?